server/client.go

```diff
@@ -26,8 +26,10 @@ import (
 )
 
 const autocompleteSearchRoute = "2/jql/autocompletedata/suggestions"
+const commentVisibilityRoute = "2/user"
```

Comment:
I'm thinking we want to use [project roles](https://developer.atlassian.com/cloud/jira/platform/rest/v2/api-group-project-roles/#api-rest-api-2-project-projectidorkey-role-get) for this

I have a WIP here that implements the data fetching piece, but not the comment filtering piece https://github.com/mattermost/mattermost-plugin-jira/compare/master...comment-security

https://github.com/mattermost/mattermost-plugin-jira/blob/a86928d1d0c0c2493df5a2690ea8f54f515fd9a5/server/autocomplete_search.go#L108-L121

Reply:
@mickmister We are getting the user `groups` here. I don't think the project roles API returns that response